### ISSUE TITLE
biosample error

### ISSUE THREAD

**Issue by PAMorin** [pachterlab/ffq#48]
I tried to install using the instructions below, but the "devel" part caused an error:
pip install git+https://github.com/pachterlab/ffq.git@devel

WARNING: Did not find branch or tag 'devel', assuming revision or ref.
  Running command git checkout -q devel
  error: pathspec 'devel' did not match any file(s) known to git
  error: subprocess-exited-with-error

I went to your githup page and installed the current version using the command shown there:

pip install git+https://github.com/pachterlab/ffq

That worked, and one of the biosample numbers I was trying to get data from worked (SAMN16895766), but a couple of others that I tried still return errors, e.g.;

ffq SAMN12326775 > Ttru_biosample_SAMN12326775_data.txt
[2022-08-10 11:42:48,468]   ERROR 404 Client Error:  for url: https://www.ebi.ac.uk/ena/browser/api/xml/SAMN12326775/
[2022-08-10 11:42:48,468]   ERROR Provided accession is invalid

This accession (https://www.ncbi.nlm.nih.gov/biosample?Db=biosample&DbFrom=bioproject&Cmd=Link&LinkName=bioproject_biosample&LinkReadableName=BioSample&ordinalpos=1&IdsFromResult=625792) is valid. Is this still an issue with not using the current development version of ffq, or a different issue?

**Comment by sbooeshaghi** [comment]
Hi,

Regarding installation, we recently released a new `ffq` version 0.3.0 and have deleted the `devel` branch as all of the changes are now in `master`. 

Regarding SAMN accessions:

NCBI has told us that there is a bug in their API when retrieving biosample accessions, that are prefixed with SAMEXXX, that they have not fixed yet. For that reason, we have opted to use the ENA api for biosample accessions. There seems to be some discrepancies between the SRA and ENA and that is why metadata may sometimes not be found for some accessions and other's have run into this problem before (see the discussion here: https://github.com/pachterlab/ffq/issues/23).

In the meantime we will look into measures to enable biosample fetching within `ffq` that is aware of the ENA<->SRA synching and the NCBI bug.

I hope that clarifies things!
